src/main/java/org/matsim/run/RunCalibration.java

```diff
@@ -312,7 +310,7 @@ private static PtFaresConfigGroup setupPTFaresGroup() {
     private static SBBTransitConfigGroup setupSBBTransit() {
 
         SBBTransitConfigGroup sbbTransit = new SBBTransitConfigGroup();
-        Set<String> modes = new HashSet<>(Arrays.asList(new String[]{"train", "tram", "bus"}));
```

Comment:
Here you can use
`var modes = Set.of("train", ....);`

I don't know whether we have those as constants, but for many modes, we have constants:
`var mdoes = Set.of(TransportMode.train, TransportMode.bus, ....);`

Reply:
PrepareTransitSchedule had a similar code section which was also changed accordingly.